crates/astria-tracing-tower/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+/// A vendored copy of the unpublished [`tracing-tower`] crate:
```

Comment:
are there differences between this and penumbra's crate?

Reply:
none, using upstream